src/core/layers/sightings.py

```diff
@@ -101,6 +103,7 @@ def _init_widgets(self, layer: QgsVectorLayer) -> None:
         idx = layer.fields().indexFromName("age")
         cfg = {}
         cfg["map"] = [
+            {"NULL": NULL},
```

Comment:
What is the meaning of `<NULL>` vs `NULL` ?

Reply:
I put `<NULL>` when it should not be able to set a null value. Here for `age`, a null value is accepted.

QGIS displays values that are not in the configuration map values with herringbones, so I do the same to simulate this behaviour. In fact, I just added `<NULL>` value to keep null at the first line of the combobox, otherwise user should use up-arrow to access value from the last one... not so user friendly in my opinion